AGENTS.md

```diff
@@ -0,0 +1,59 @@
+# Cockpit starter-kit - Technical Overview
+
+This document provides a technical overview for applications or plugins that integrate into Cockpit and are prepared with the starter-kit. Is is human readable but intended to give AI context to follow the project standards.
+
+## Project Structure
+
+- **Upstream Website**: [Cockpit Project](https://cockpit-project.org/)
+- **Upstream Code Repository**: [Cockpit Project Git](https://github.com/cockpit-project/)
+- **Upstream Starter Kit**: [starter-kit](https://github.com/cockpit-project/starter-kit)
+- **Language**: [TypeScript](https://www.typescriptlang.org/)
+- **Design System**: [Patternfly](https://www.patternfly.org)
```

Comment:
Might be different for some forked projects. This is an useful link, but better belongs into README.md for human devs.